src/System.Private.ServiceModel/tests/Scenarios/Client/ExpectedExceptions/ExpectedExceptionTests.cs

```diff
@@ -152,7 +154,7 @@ public static void SendTimeout_For_Long_Running_Operation_Throws_TimeoutExceptio
             {
                 IWcfService proxy = factory.CreateChannel();
                 watch.Start();
-                proxy.EchoWithTimeout("Hello");
+                proxy.EchoWithTimeout("Hello", serviceOperationTimeout);
             });
         }
         finally
```

Comment:
The HttpClient doesn't get passed a timeout, it get's passed a CancellationToken in the SendAsync api call. A callback is then registered against the CT which gets executed when the timeout occurs. This won't be an issue in HttpClient, it might be an issue with CancellationTokenSource or Timer. I suspect they are doing some kind of timer coalescence where some other timer has fired and then they check if there's anything else due to fire soon. Considering the standard Timer resolution is 15ms, being 8ms early is within the range of Timer accuracy.